src/client/app/components/admin/AdminComponent.tsx

```diff
@@ -0,0 +1,37 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react';
+import HeaderContainer from '../../containers/HeaderContainer';
+import FooterComponent from '../FooterComponent';
+import AddMeterContainer from '../../containers/admin/AddMetersContainer';
```

Comment:
This should be: `import AddMetersContainer from '../../containers/admin/AddMetersContainer';`